targets/TARGET_STM/trng_api.c

```diff
@@ -128,8 +128,8 @@ void trng_init(trng_t *obj)
     }
 
     /* first random number generated after setting the RNGEN bit should not be used */
-    if (HAL_RNG_GenerateRandomNumber(&obj->handle, &dummy) != HAL_OK) {
-        printf("trng_init: HAL_RNG_GenerateRandomNumber error 0x%x\n", obj->handle.ErrorCode); // should not occur
+    while (HAL_RNG_GenerateRandomNumber(&obj->handle, &dummy) != HAL_OK) {
+//        printf("trng_init: HAL_RNG_GenerateRandomNumber error 0x%x\n", obj->handle.ErrorCode);
```

Comment:
Either remove or propagate the error

Reply:
The comment should be updated as the while changes it a bit - we are waiting until we get a number

Reply:
@0xc0170 done